app/kubernetes-ingress-controller/2.0.x/guides/prometheus-grafana.md

```diff
@@ -2,21 +2,18 @@
 title: Integrate the Kubernetes Ingress Controller with Prometheus/Grafana
 ---
 
-The {{site.kic_product_name}} can give you visibility not only into how Kong is
-performing but also gives visibilty into
-how the services in your Kubernetes cluster are responding to the
-inbound traffic.
+The {{site.kic_product_name}} can give you visibility into how {{site.base_gateway}} is performing and how the services in your Kubernetes cluster are responding to the inbound traffic.
 
-This how-to guide walks through the steps of how to configure Kong and
-Prometheus to collect metrics from Kong Gateway.
+This guide below walks you through setting up monitoring for
```

Comment:
Could I please request removal of "below"? I know we have "above" and "below" EVERYWHERE but it's redundant (and especially here where "this guide" takes care of it).